crates/rune-macros/src/context.rs

```diff
@@ -493,6 +497,55 @@ impl Context {
 
                         attrs.install_with = Some(install_with);
                     }
+                    Meta(NameValue(syn::MetaNameValue {
+                        path,
+                        lit: Lit::Str(s),
+                        ..
+                    })) if path == DYNAMIC_FIELDS => {
+                        attrs.meta_fields = Some(match s.value().as_str() {
+                            "never" => format_ident!("Never"),
+                            "only" => {
+                                if cfg!(feature = "dynamic_fields") {
```

Comment:
Since this kind of test is repeated three times it might be worth breaking out into a function.